integration/maptest/map.go

```diff
@@ -277,25 +279,37 @@ func RunLeafHistory(ctx context.Context, t *testing.T, tadmin trillian.TrillianA
 
 			for _, batch := range tc.get {
 				indexes := [][]byte{batch.Index}
-				getResp, err := tmap.GetLeaves(ctx, &trillian.GetMapLeavesRequest{
+				getResp, err := tmap.GetLeavesByRevision(ctx, &trillian.GetMapLeavesByRevisionRequest{
 					MapId:    tree.TreeId,
 					Index:    indexes,
 					Revision: batch.revision,
 				})
+				switch batch.revision {
```

Comment:
I wonder if this should be a separate test really. 
TestHistory is verifying that generic set and get operations work correctly. This new bit is testing an edge case around GetLeaves?

Reply:
Yep, happy to split it out.